apps/platform/trpc/routers/userRouter/securityRouter.ts

```diff
@@ -1081,5 +1115,226 @@ export const securityRouter = router({
+          .delete(emailIdentities)
+          .where(inArray(emailIdentities.orgId, orgIdsArray)),
+        db
+          .delete(emailIdentitiesAuthorizedOrgMembers)
+          .where(
+            inArray(emailIdentitiesAuthorizedOrgMembers.orgId, orgIdsArray)
+          ),
+        db
+          .delete(emailIdentitiesPersonal)
+          .where(inArray(emailIdentitiesPersonal.orgId, orgIdsArray)),
+        db
+          .delete(emailIdentityExternal)
+          .where(inArray(emailIdentityExternal.orgId, orgIdsArray)),
+        db.delete(convos).where(inArray(convos.orgId, orgIdsArray)),
+        db
+          .delete(convoSubjects)
+          .where(inArray(convoSubjects.orgId, orgIdsArray)),
+        db
+          .delete(convoParticipants)
+          .where(inArray(convoParticipants.orgId, orgIdsArray)),
+        db
+          .delete(convoParticipantTeamMembers)
+          .where(inArray(convoParticipantTeamMembers.orgId, orgIdsArray)),
+        db
+          .delete(convoAttachments)
+          .where(inArray(convoAttachments.orgId, orgIdsArray)),
+        db
+          .delete(pendingAttachments)
+          .where(inArray(pendingAttachments.orgId, orgIdsArray)),
+        db.delete(convoEntries).where(inArray(convoEntries.orgId, orgIdsArray)),
+        db
+          .delete(convoEntryReplies)
+          .where(inArray(convoEntryReplies.orgId, orgIdsArray)),
+        db
+          .delete(convoEntryPrivateVisibilityParticipants)
+          .where(
+            inArray(convoEntryPrivateVisibilityParticipants.orgId, orgIdsArray)
+          ),
+        db
+          .delete(convoEntryRawHtmlEmails)
+          .where(inArray(convoEntryRawHtmlEmails.orgId, orgIdsArray)),
+        db
+          .delete(convoSeenTimestamps)
+          .where(inArray(convoSeenTimestamps.orgId, orgIdsArray)),
+        db
+          .delete(convoEntrySeenTimestamps)
+          .where(inArray(convoEntrySeenTimestamps.orgId, orgIdsArray))
+      ]);
+
+      // Delete attachments
+
+      const orgPublicIdsArray = orgsQuery.map((org) => org.publicId);
+
+      const deleteStorageResponse = (await fetch(
+        `${env.STORAGE_URL}/api/orgs/delete`,
+        {
+          method: 'post',
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: env.STORAGE_KEY
+          },
+          body: JSON.stringify({
+            orgPublicIds: orgPublicIdsArray
+          })
+        }
+      ).then((res) => res.json())) as unknown;
+
+      if (!deleteStorageResponse) {
+        console.error('🔥 Failed to delete attachments from storage', {
+          orgPublicIdsArray
+        });
+      }
+
+      // Delete Billing
+
+      if (env.EE_LICENSE_KEY) {
+        await Promise.all(
+          orgIdsArray.map(async (orgId) => {
+            await billingTrpcClient.stripe.subscriptions.cancelOrgSubscription.mutate(
+              {
+                orgId: orgId
+              }
+            );
+          })
+        );
+      }
+    }
+
```

Comment:
we need to delete postal orgs too when deleting the orgs

Reply:
yeah, no idea how to do that with the current postal setup
can add it later

Reply:
Ok I would handle that in a diff PR